Tests/IPAddressTests/IPAddressTests.cs

```diff
@@ -301,5 +301,24 @@ public void NetTest6_SocketAddressBasic()
                         + " as " + socketAddress2.ToString() + " " + socketAddress2.GetHashCode());
             }
         }
+
+        [TestMethod]
+        public void Equality_Tests()
+        {
+            var privateAddress = new IPAddress(new byte[] { 192, 168, 0, 1 });
+            var publicAddress = new IPAddress(new byte[] { 1, 1, 1, 1 });
+
+            // Equal
+            Assert.IsTrue(privateAddress.Equals(new IPAddress(new byte[]{192, 168, 0 , 1})), "192.168.0.1 equals 192.168.0.1");
+            Assert.IsTrue(publicAddress.Equals(new IPAddress(new byte[] { 1, 1, 1, 1 })), "1.1.1.1 equals 1.1.1.1");
```

Comment:
The comment you've made are rather similar to the ones that the test framework produces by default. Just mentioning it so you don't have all this trouble in future work.

Reply:
I think I added these because `IsTrue` doesn't have a message by default. Although in this case I could have used `AreEqual` as I actually want to use the `Equals` method (I think it was copy/paste from the `==` cases). 

Will update.